ic-os/guestos/defs.bzl

```diff
@@ -110,6 +110,10 @@ def image_deps(mode, malicious = False):
         # Dev guest_upgrade client
         deps["rootfs"].pop("//rs/ic_os/guest_upgrade/client", None)
         deps["rootfs"].update({"//rs/ic_os/guest_upgrade/client:client_dev": "/opt/ic/bin/guest_upgrade_client:0755"})
+    else:
+        deps["component_files"].update({
+            Label("//ic-os/components:misc/serial-getty@/guestos-prod/override.conf"): "/etc/systemd/system/serial-getty@.service.d/override.conf",
```

Comment:
tiny nit:

This _could_ be used as the default in components instead. It would make it a tiny bit harder to break, but in the end, if we do miss the override the worst thing is a login prompt with no passwords.